src/java.net.http/share/classes/java/net/http/HttpResponse.java

```diff
@@ -272,7 +276,8 @@ public interface ResponseInfo {
      * <p>The following are examples of using the predefined body handlers to
      * convert a flow of response body data into common high-level Java objects:
      *
-     * <pre>{@code    // Receives the response body as a String
+     * {@snippet :
```

Comment:
Same remark here. It would make more sense to have each example in its own snippet.